Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -108,7 +108,8 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
 
 3. Paste the above output from `create-external-cluster-resources.py` into your current shell to allow importing the source data.
 
-4. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
+4. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.  Note that if your Rook cluster nodes are running
+   a kernel earlier than 5.4 or equivalent you may need to remove `fast-diff,image-map,deep-flatten,exclusive-lock` from the `imageFeatures` line.
```

Comment:
Do we have an `image-map` image feature? it should be `object-map` isn't it?

Reply:
My ignominy is without bound. You're right, thanks.  Doing too many things at once with too many distractions :-/  This is why we have PR review.